client/my-sites/stats/activity-log-banner/upgrade-banner.jsx

```diff
@@ -11,29 +11,54 @@ import { localize } from 'i18n-calypso';
  */
 import { isJetpackSite } from 'state/sites/selectors';
 import Banner from 'components/banner';
-import { PLAN_PERSONAL, FEATURE_JETPACK_ESSENTIAL } from 'lib/plans/constants';
+import {
+	FEATURE_JETPACK_ESSENTIAL,
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+	PLAN_JETPACK_PERSONAL_MONTHLY,
+	PLAN_PERSONAL,
+} from 'lib/plans/constants';
 
 class UpgradeBanner extends Component {
 	render() {
 		const { translate, isJetpack } = this.props;
-		if ( isJetpack ) {
-			return null;
-		}
-		return (
+		return isJetpack ? (
 			<div className="activity-log-banner__upgrade">
 				<Banner
-					callToAction={ translate( 'More Details' ) }
-					dismissPreferenceName="activity-upgrade-banner-simple"
+					callToAction={ translate( 'Learn more' ) }
+					dismissPreferenceName="activity-upgrade-banner-jetpack"
+					event="activity_log_upgrade_click_wpcom"
+					feature={ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY }
+					plan={ PLAN_JETPACK_PERSONAL_MONTHLY }
+					title={ translate( "Upgrade to a plan to access your site's complete log" ) }
+					description={ translate(
+						"With your free plan, your site's activity log will only display the last 20 events."
+					) }
+					list={ [
+						translate( 'Full activity for the past 30 days' ),
+						translate( 'Daily automated backups and spam filtering' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Daily automated backups and bulletproof spam filtering' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).